CIPs/0022.md

```diff
@@ -0,0 +1,21 @@
+# CIP [0022]: Standard for Injacted web3 Provider
+
+- Date: 2020-10-29
+- Author: @daoauth @NAKsir-melody
```

Comment:
hi @daoauth  thank you for submitting this CIP, I would like you to update it to follow the newly activated CIP process which has it's newly updated template here: https://github.com/celo-org/celo-proposals/blob/master/CIPs/cip-template.md

Also, if the CIP is in Draft status, I'd be able to merge the proposal into the repo.

Reply:
@YazzyYaz I updated CIP, But the number is wrong, I registered first, but another CIP is using it first. It is suggested to use the pull number. (Like EIP)